api/api-javascript/api-database/src/main/resources/META-INF/dirigible/db/v4/query.js

```diff
@@ -19,17 +19,9 @@ exports.execute = function(sql, parameters, databaseType, datasourceName) {
 	let resultset = [];
 	if (parameters) {
 		const params = JSON.stringify(parameters);
-		if (databaseType) {
-			if (datasourceName) {
-				resultset = org.eclipse.dirigible.api.v3.db.DatabaseFacade.query(sql, params, databaseType, datasourceName);
-			} else {
-				resultset = org.eclipse.dirigible.api.v3.db.DatabaseFacade.query(sql, params, databaseType);
-			}
-		} else {
-			resultset = org.eclipse.dirigible.api.v3.db.DatabaseFacade.query(sql, params);
-		}
+		resultset = org.eclipse.dirigible.api.v3.db.DatabaseFacade.query(sql,params,databaseType,datasourceName);
```

Comment:
Please double check what would happen, if the API is used in the following way:

```javascript
query.execute("SELECT * FROM DUMMY");
```

and also

```javascript
query.execute("SELECT * FROM DUMMY WHERE NAME = ?", ["Test123"]);
```

Then the connection should be to the default configured datasource connection (e.g. local/DefaultDB when no environment variables are provided)

Reply:
Checked. Works as expected.